docroot/modules/custom/yukon_w3_custom/templates/edit-language.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if items is not empty %}
+    <ul class="tabs--secondary pagination pagination-sm">
+       <li class="{% if language == "en" %}active{% endif %}">
+           <a href="/node/{{ items }}/edit" class="language-link " xml:lang="en">English</a></li>
```

Comment:
Links are generated in a wrong way.
Check the logic of 'Translate' page. The link for the language can't always be "edit" if it doesn't exist. So check the Drupal implementation of that page and reuse the logic, please.
![image](https://github.com/ytgov/yukon-ca/assets/123640502/5bdb902a-6a95-43c0-96d3-a6d126a53cc4)

Reply:
The translation page button text has been updated for missing pages.